gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/modules/image-resolution/image-resolver.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { getImageUrl } from "./image-resolver";
+
+describe('image-resolver', () => {
+    describe('getImageUrl', () => {
+        it('should generate a URL for an unknown image based on the default config', () => {
+            // NOTE: this test basically validates that we have no guard clause for "reasonable" cards
```

Comment:
this bugs me a bit. We didn't have anything in the existing code to stop somebody from passing an invalid set #... so I wrote a test to prove that still works